examples/check_integration_error.py

```diff
@@ -6,123 +6,131 @@
 × t2 × vd × repeats) w/ normally distributed random noise.
 Check that the following match:
 
-- integral w/ error (the canned routine :func:`~pyspecProcScripts.integral_w_errors`)
+- integral w/ error (the canned routine
+  :func:`~pyspecProcScripts.integral_w_errors`)
 - propagate error based off the programmed σ of the normal distribution
 - set the error bars based on the standard deviation (along the repeats
   dimension) of the *real* part of the integral
-- propagate error based off the variance of the noise in the inactive
-  coherence channels (do this manually inside this script -- should mimic
-  what :func:`~pyspecProcScripts.integral_w_errors` does)
+- propagate error based off the variance of the noise in the inactive coherence
+  channels (do this manually inside this script -- should mimic what
+  :func:`~pyspecProcScripts.integral_w_errors` does)
 """
-from pylab import *
-from pyspecdata import *
-from pyspecProcScripts import integrate_limits, integral_w_errors
+import pyspecdata as psd
+import numpy as np
+from numpy import r_
+from pyspecProcScripts import integral_w_errors, select_pathway
+from pylab import rcParams
+import sympy as s
+from collections import OrderedDict
 
+rcParams["image.aspect"] = "auto"  # for sphinx gallery
 # sphinx_gallery_thumbnail_number = 1
 
-init_logging(level="debug")
-fl = figlist_var()
-t2 = nddata(r_[0:1:1024j], "t2")
-vd = nddata(r_[0:1:40j], "vd")
-ph1 = nddata(r_[0, 2] / 4.0, "ph1")
-ph2 = nddata(r_[0:4] / 4.0, "ph2")
+t2, vd, ph1, ph2 = s.symbols("t2 vd ph1 ph2")
+n_repeats = 10
+noise_bounds = (0, 200)  # seem reasonable to me
 signal_pathway = {"ph1": 0, "ph2": 1}
-excluded_pathways = [(0, 0), (0, 3)]
```

Comment:
no longer needed since the "excluded pathways" for determining noise is determined with a mask

Reply:
no -- the set logic is good and should be left in place.